api/controllers/posts.js

```diff
@@ -0,0 +1,33 @@
+const Post = require("../models/Post");
+
+// trying to get all posts so that I can find out the id of each post
+async function index(req, res) {
+    try {
+        const posts = await Post.all;
```

Comment:
Was wondering if it is necessary to make an all function given its not used in your website?